### ISSUE TITLE
[BUG] 

### ISSUE THREAD

**Issue by nazarov-ae** [PyCQA/prospector#570]
Hi!
**Describe the bug**
Prospector raised exception at run
```
  File ".../prospector/run.py", line 112, in execute
    raise FatalProspectorException from ex
TypeError: __init__() missing 1 required positional argument: 'message'
```

**To Reproduce**
```
1. pip install prospector==1.7.7
2. pip install pylint (currently 2.15.9)
3. prospector . -X
```

**Expected behavior** A clear and concise description of what you expected to happen.

**Environment:**
- Linux 5.4.0-135-generic
- pylint 2.15.9
- prospector 1.7.7
- python 3.8.3

**Additional context**
Freeze prospector 1.7.7 for avoid https://github.com/PyCQA/prospector/issues/557

**Comment by Pierre-Sassoulas** [comment]
This is fixed in 1.8.3, please upgrade :)

**Comment by nazarov-ae** [comment]
> This is fixed in 1.8.3, please upgrade :)

Thanks!
But after upgrade appear this bug https://github.com/PyCQA/prospector/issues/557

**Comment by teijo** [comment]
`1.8.x` also introduces Git dependency bug https://github.com/PyCQA/prospector/issues/556 which blocks our upgrade.

I manged to pin `pylint = 2.15.6` which was a transient dependency for me. `pylint = 2.15.7` crashes.

My workaround for now (FYI @nazarov-ae):
```
$ pylint --version
pylint 2.15.6
astroid 2.12.13
Python 3.9.13 (main, Jun 24 2022, 15:22:58)
[Clang 13.1.6 (clang-1316.0.21.2.3)]
$ prospector --version
prospector 1.7.7
```

`pyproject.toml`
```toml
prospector = "1.7.7"
pylint = "2.15.6"
```

Just heads-up that if you pin to this version (and not and older one) and use Django `Q` operators, you'll encounter this open bug https://github.com/PyCQA/pylint/issues/7381. Can be bypassed with `# pylint: disable=unsupported-binary-operation
` case-by-case if needed.
